modules/widgets/top-posts.php

```diff
@@ -488,12 +488,14 @@ function get_by_views( $count, $args ) {
 			$days = 2;
 		}
 
-		$post_view_posts = stats_get_csv( 'postviews', array( 'days' => absint( $days ), 'limit' => 11 ) );
-		if ( ! $post_view_posts ) {
+		$post_view_posts = stats_get_from_restapi( array(), 'top-posts?max=10&summarize=1&num=' . absint( $days ) );
```

Comment:
Nice!

Did you add the query parameters to the sub-endpoint URL instead of the arguments array to bypass #5860?

Also, is there a reason why you went with `10` instead of `11`? in this case?